### PR TITLE
docs(contributing): sync up maintainer lgtm policy

### PR DESCRIPTION
**Pull request by mboersma**

### CODE REVIEW

**Comment by krancour** [comment]
LGTM

File: docs/contributing/standards.rst [hunk 1]
```diff
@@ -169,6 +169,10 @@ to indicate that a PR is acceptable. Any code change--other than
 a simple typo fix or one-line documentation change--requires at least two
 maintainers to accept it.
 
+No pull requests can be merged until at least one core maintainer signs off
```
Comment (carmstrong) [inline docs/contributing/standards.rst]: Should we link `maintainer` here to the root MAINTAINERS.md?

**Comment by carmstrong** [comment]
LGTM with or without my comment.

File: docs/contributing/standards.rst (hunk 1 above)
Comment (bacongobbler) [inline docs/contributing/standards.rst]: I would tend to agree since we don't have any official documentation describing a maintainer's role.

**Comment by mboersma** [comment]
This passed the relevant documentation tests, so I'm merging it.

**Pull request merged**